docker/build-tools/Dockerfile

```diff
@@ -80,10 +73,14 @@ RUN apt-get purge -y \
     zlib1g-dev \
     && apt-get autoremove -y \
     && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -fr /home /tmp/* \
+    && go clean -cache \
+    && go clean -modcache
 
-# TODO: Remove this if the FROM distroless below is made to work.
-RUN rm -fr /root/.cache/go-build /home /tmp/*
+# Setup to run the Go compiler with host-side build cache, test cache, and module cache
+ENV GOCACHE=/gocache
+RUN rm -fr /go/pkg/mod
```

Comment:
Is there a reason to remove this directory as you already emptied it above with the `clean -modcache`?

Reply:
When running the container, I need to mount the host's module cache at that location, so it seemed safer to remove the directory outright. 

I explicitly empty the directory previously using go clean -modcache just in case  that command decides to clean something else up that might now be in that same directory. It's likely 100% redundant, but it also doesn't hurt...